cmd/telemetrygen/pkg/metrics/config.go

```diff
@@ -16,26 +16,44 @@ type Config struct {
 	common.Config
 	NumMetrics int
 	MetricName string
-	MetricType metricType
+	MetricType MetricType
```

Comment:
Why do we need to export this type?

Reply:
Since we're using a locally defined variable type here, I exported it so it's easy to adjust the type of Metrics being generated using the defined constants like so:


```go
mc := metrics.NewConfig()
mc.Endpoint = "localhost:1234"
mc.RunDuration = 5 * time.Second
mc.MetricType = metrics.MetricTypeHistogram
err := metrics.Start(mc)
```

Reply:
Okay, makes sense!